integration-tests/conftest.py

```diff
@@ -34,3 +35,26 @@ def register_subman(
             password=test_config.get("candlepin", "password"),
         )
     yield subman_session
+
+
+"""
+An helper function to handle a time periond waiting for an external service
+to update its state.
+
+an example:
+
+   result = loop_until(lambda: insights_client.is_registered)
+   assert result == True
+
+   The loop function will retry to run predicate every 5secs
+   untill the total time exceeds timeout_sec.
+"""
```

Comment:
This docstring should be moved inside the function under the `def` line, that way `help(conftest.loop_until)` will work.  See: https://peps.python.org/pep-0257/